modules/odc-creating-projects-using-developer-perspective.adoc

```diff
@@ -4,19 +4,23 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="odc-creating-projects-using-developer-perspective_{context}"]
-= Creating a project using the Developer perspective in the web console
+= Creating a project by using the Developer perspective in the web console
 
 You can use the *Developer* perspective in the {product-title} web console to create a project in your cluster.
 
 [NOTE]
 ====
+<<<<<<< HEAD
```

Comment:
This is left over from a merge conflict.

Reply:
Thanks! I've been staring at the files so long, I think I've gone blind to them!

Reply:
It happens!